### PR TITLE
Replace SCT Course 2020 iCloud link with Google Drive link

### LINKED ISSUE #3613: Replace iCloud links with links to Google Drive-hosted PDF files
Our docs currently contain iCloud links on our [Tutorials and Courses](https://spinalcordtoolbox.com/user_section/tutorials.html) page. But, iCloud is slow and awkward to use compared to a PDF file, so we should export to PDF then link that file instead.

### PR DESCRIPTION
**Pull request by joshuacwnewton**
#### GitHub

- [ ] I've given this PR a concise, self-descriptive, and meaningful title
- [ ] I've linked relevant issues in the PR body
- [ ] I've applied [the relevant labels](https://www.neuro.polymtl.ca/software/contributing#pr_labels) to this PR
- [ ] I've applied a [release milestone](https://github.com/spinalcordtoolbox/spinalcordtoolbox/milestones) (major, minor, patch) in line with [Semantic Versioning guidelines](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Misc%3A-Creating-a-new-release#convention-for-naming-releases) 
- [ ] I've assigned a reviewer

#### PR contents

- [ ] I've consulted [SCT's internal developer documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki) to ensure my contribution is in line with any relevant design decisions
- [ ] I've added [relevant tests](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Tests) for my contribution
- [ ] I've updated the [relevant documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages

## Description

All I've done is export the slides from [this iCloud link](https://www.icloud.com/keynote/0th8lcatyVPkM_W14zpjynr5g#SCT%5FCourse%5F20200121) then uploaded the PDF to [this Google Drive folder](https://drive.google.com/drive/u/0/folders/1QBjDxjTUDZjf3Yxy9bgxoMfHsGyHLXXQ).

## Linked issues

Fixes #3613.

### CODE REVIEW

File: documentation/source/user_section/tutorials.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ The past courses handouts are listed below:
 -  `SCT course (v3.0_beta1), Montreal, 2016-04-19`_
 -  `SCT Hands-on Workshop (v2.0.4), Toronto, 2015-06-15`_
 
-.. _SCT course (v4.2.1), London, 2020-01-21: https://www.icloud.com/keynote/0th8lcatyVPkM_W14zpjynr5g#SCT%5FCourse%5F20200121
+.. _SCT course (v4.2.1), London, 2020-01-21: https://drive.google.com/file/d/1rn6srrQRLmO6cMHJ9OSrOBD7xjMm6uzn/view?usp=sharing
```
Comment (joshuacwnewton) [inline documentation/source/user_section/tutorials.rst:72]: Note: This link includes slides only. If we want to include presenter notes, someone with Keynote on their Mac will need to export the slides with presenter notes. (I couldn't figure out how to do this via iCloud only.)

Comment (jcohenadad) [reply documentation/source/user_section/tutorials.rst:72]: I'm happy to do that once the little issues in the slides are fixed

Comment (joshuacwnewton) [reply documentation/source/user_section/tutorials.rst:72]: Ah! So, this is the link to the 2020 slides (i.e. last year's). So, this one is fine to export now -- my plan was to add the 2021 slides will be added in a follow-up PR.

Comment (jcohenadad) [reply documentation/source/user_section/tutorials.rst:72]: here you go:
https://drive.google.com/file/d/1TZireJ6yhV8q7PbKKXyXg7Heov9-YJMu/view?usp=sharing

Comment (jcohenadad) [reply documentation/source/user_section/tutorials.rst:72]: (also added in the same gdrive folder)

**Comment by joshuacwnewton** [comment]
Merging without review, as this is a tiny change proposed by @jcohenadad himself, and I don't want to waste his time in asking for an approval on his own changes. :P

**Pull request merged**